hangman.js

```diff
@@ -1 +1,51 @@
-console.log("welcome to hangman");
\ No newline at end of file
+players = [];
+word = [];
+userattempts = 0;
+var win = 0;
+var lose = 0;
+// Prompt welcomes users to game:
+alert ("Welcome to Hangman. Let's Play!")
+
+// Player Names are entered
+
+/*function users(){
+    var playerName = prompt(players.length + " player(s) so far. Enter a player name, or click 'cancel' to play.");
+    if(playerName){
+      players.push(playerName);
+      users();
+    }
+  }
+*/
+
+// Secret word is chosen:
+function pickAWord () {
+    var secretWord = prompt ("Please enter your secret word:");
+    if(secretWord){
+      word = secretWord.split("");
+    }
+  }
```

Comment:
:+1: for making this its own little function! Generally, though, functions should `return` something.